atomic

```diff
@@ -514,6 +518,18 @@ def create_parser(help_text):
                      help=_("Only display container IDs"))
     add_opt(pss)
 
+    # atomic sign
+    signer = ATOMIC_CONFIG.get('default_signer', None)
+    sig_store = ATOMIC_CONFIG.get('default-sigstore-path', "/var/lib/atomic/containers")
+    signp = subparser.add_parser("sign", help="Sign an image",
+                                 epilog="Create a signature for an image which can be used later to verify it.")
+    signp.set_defaults(_class=Sign, func="sign")
+    signp.add_argument("images", nargs="*", help=_("images to sign"))
+    signp.add_argument("--sign_key", dest="sign_key", default=signer,
+                       help=_("Name of the signing key. Currently %s, default can be defined in /etc/atomic.conf" % signer))
```

Comment:
`--sign_key` reads a bit like “sign the specified key”. `--signing_key`, `--sign_by`?

Also, other options seem to use `-`, not `_`, to separate words.